components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -392,7 +394,7 @@
             <%
                 if (description != null && description.trim().length() > 0) {
             %>
-            <td><textarea name="description" id="description" class="text-box-big"><%=description%>
+            <td><textarea name="description" id="description" class="text-box-big"><%=Encode.forHtml(description)%>
             </textarea>
```

Comment:
The exact method to be used here is Encode.forHtmlContent(). Using Encode.forHtml() is not wrong, but that will also encode characters which are not significant in this context.